lib/duration.ts

```diff
@@ -292,7 +292,8 @@ export class Duration implements Temporal.Duration {
       microseconds,
       nanoseconds
     );
-    const options = ES.GetOptionsObject(optionsParam);
+    const options =
+      typeof optionsParam === 'string' ? { smallestUnit: optionsParam } : ES.GetOptionsObject(optionsParam);
```

Comment:
I think we need to change this, I left a comment on the proposal-temporal PR: https://github.com/tc39/proposal-temporal/pull/1875/files#r748524136

Reply:
Fixed!  To reduce bundle size and to make the TS part easier I implemented the change slightly differently here than in proposal-temporal, but result is the same.